llpc/translator/include/LLVMSPIRVLib.h

```diff
@@ -86,6 +86,14 @@ struct SPIRVSpecConstEntry
 /// \brief Represents the map from SpecId to specialization constant data.
 typedef std::map<uint32_t, SPIRVSpecConstEntry> SPIRVSpecConstMap;
 
+// A converting sampler with a constant value (or indexable array thereof).
+struct ConvertingSampler {
+  unsigned set;                    // Descriptor set
+  unsigned binding;                // Binding
+  llvm::ArrayRef<uint32_t> values; // Values; 8 uint32_t per array entry
+};
+static const unsigned ConvertingSamplerDwordSize = 8;
```

Comment:
Perhaps rename this to `ConvertingSamplerDwordCount`? I got confused by the DwordSize part initially.